composer.json

```diff
@@ -19,54 +19,54 @@
     ],
     "require": {
         "php": "^8.0",
-        "calebporzio/sushi": "^2.1",
-        "danharrin/livewire-rate-limiting": "^0.3.0",
-        "ergebnis/phpstan-rules": "^0.15.3",
-        "fakerphp/faker": "^1.15",
-        "friendsofphp/php-cs-fixer": "^3.0",
+        "calebporzio/sushi": "^2.3",
+        "danharrin/livewire-rate-limiting": "^0.3",
+        "ergebnis/phpstan-rules": "^0.15",
+        "fakerphp/faker": "^1.16",
+        "friendsofphp/php-cs-fixer": "^3.1",
         "google/recaptcha": "^1.2",
         "graham-campbell/analyzer": "^3.0",
         "guzzlehttp/guzzle": "^7.3",
-        "illuminate/bus": "^8.0",
-        "illuminate/contracts": "^8.0",
-        "illuminate/http": "^8.0",
-        "illuminate/mail": "^8.0",
-        "illuminate/support": "^8.0",
-        "illuminate/view": "^8.0",
+        "illuminate/bus": "^8.62",
+        "illuminate/contracts": "^8.62",
+        "illuminate/http": "^8.62",
+        "illuminate/mail": "^8.62",
+        "illuminate/support": "^8.62",
+        "illuminate/view": "^8.62",
         "jasonmccreary/laravel-test-assertions": "^2.0",
         "jenssegers/agent": "^2.6",
         "johnkary/phpunit-speedtrap": "^4.0",
         "laravel/fortify": "^1.8",
         "league/commonmark": "^2.0",
         "livewire/livewire": "^2.6",
         "mockery/mockery": "^1.4",
-        "nunomaduro/collision": "^5.5",
+        "nunomaduro/collision": "^5.10",
         "nunomaduro/larastan": "^0.7",
         "nunomaduro/laravel-mojito": "^0.2",
-        "orchestra/testbench": "^6.9",
-        "pestphp/pest": "^1.11",
+        "orchestra/testbench": "^6.21",
+        "pestphp/pest": "^1.20",
         "pestphp/pest-plugin-faker": "^1.0",
         "pestphp/pest-plugin-laravel": "^1.1",
         "pestphp/pest-plugin-livewire": "^1.0",
-        "pestphp/pest-plugin-parallel": "^0.3.1",
+        "pestphp/pest-plugin-parallel": "^0.3",
         "phpstan/extension-installer": "^1.1",
-        "phpstan/phpstan-deprecation-rules": "^0.12.6",
-        "phpstan/phpstan-strict-rules": "^0.12.10",
+        "phpstan/phpstan-deprecation-rules": "^0.12",
+        "phpstan/phpstan-strict-rules": "^0.12",
         "phpunit/phpunit": "^9.5",
-        "pragmarx/google2fa-laravel": "^1.3",
-        "rector/rector": "^0.11.38",
+        "pragmarx/google2fa-laravel": "^2.0",
+        "rector/rector": "^0.11",
         "ruafozy/mersenne-twister": "^1.3",
-        "spatie/laravel-flash": "^1.7",
+        "spatie/laravel-flash": "^1.8",
         "spatie/laravel-honeypot": "^4.0",
         "spatie/laravel-medialibrary": "^9.7",
         "spatie/laravel-newsletter": "^4.10",
         "spatie/laravel-package-tools": "^1.9",
         "spatie/laravel-personal-data-export": "^3.0",
-        "spatie/laravel-schemaless-attributes": "^1.8",
+        "spatie/laravel-schemaless-attributes": "^2.0",
```

Comment:
Major

Reply:
The major release seems to be just bug fixes and PHP 8.0 compatibility.

- https://github.com/spatie/laravel-schemaless-attributes/releases/tag/2.0.0